Source/ProgressionSystemRuntime/Public/Data/PSWorldSubsystem.h

```diff
@@ -108,14 +108,18 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSWorldSubsystem : public UWorldSubsystem
 	UPROPERTY(VisibleInstanceOnly, BlueprintReadWrite, Transient, Category = "C++", meta = (BlueprintProtected, DisplayName = "Pool Actors Handlers"))
 	TArray<FPoolObjectHandle> PoolActorHandlersInternal;
 
-	/** Array of actors spawned */
-	UPROPERTY(VisibleInstanceOnly, BlueprintReadWrite, Transient, Category = "C++", meta = (BlueprintProtected, DisplayName = "Spawned Actors Internal"))
-	TArray<AActor*> SpawnedStarActorsInternal;
-
 	/** Store the material for dynamic progress materil fill for a star actor */
 	UPROPERTY(VisibleInstanceOnly, BlueprintReadWrite, Category = "C++", meta = (BlueprintProtected, DisplayName = "Star Dynamic Progress Material"))
 	TObjectPtr<UMaterialInstanceDynamic> StarDynamicProgressMaterial = nullptr;
 
+	UPROPERTY(VisibleInstanceOnly, BlueprintReadWrite, Category = "C++", meta = (BlueprintProtected, DisplayName = "Star Material Slot Divisor"))
+	FName StarMaterialSlotDivisor = TEXT("Percentage2");
```

Comment:
Sorry, it's my fault. I initially asked you to call the slot name "Divisor," but that was my copy-paste mistake. "Divisor" should only be a float value. Please remove "Divisor" from the slot name: `StarMaterialSlot`.

Reply:
moved to data asset: https://github.com/h4rdmol/ProgressionSystem/pull/17/commits/1667bc35491808340e600ea0cafa0b31dc34e408